service/controller/clusterapi/v27/templates/cloudconfig/instance_storage_class.go

```diff
@@ -1,11 +1,11 @@
 package cloudconfig
 
-const InstanceStorageClassContent = `apiVersion: storage.k8s.io/v1beta1
+const InstanceStorageClassContent = `apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: gp2
   annotations:
-    storageclass.beta.kubernetes.io/is-default-class: "true"
+    storageclass.kubernetes.io/is-default-class: "true"
```

Comment:
You probably want to do this for the legacy controller resources as well. Maybe also other versions if necessary. 

```
service/controller/legacy/v27/templates/cloudconfig/instance_storage_class.go
```

Reply:
but can I modify if they are not WIP? or it `clusterapi` no wired yet?

Reply:
v27 is WIP. And clusterapi is not wired.

Reply:
done